src/setuptools_scm/version.py

```diff
@@ -13,14 +14,34 @@
 SEMVER_MINOR = 2
 SEMVER_PATCH = 3
 SEMVER_LEN = 3
-TAG_PREFIX = re.compile(r"^\w+-(.*)")
 
 
 def _pad(iterable, size, padding=None):
     padded = chain(iterable, repeat(padding))
     return list(islice(padded, size))
 
 
+def _parse_version_tag(tag, config):
+    tagstring = tag if not isinstance(tag, string_types) else '%s' % tag
```

Comment:
please use `str(tag)` instead of `%` formatting

Reply:
done, not sure why I did it in the first place